exercises/practice/all-your-base/AllYourBase.ps1

```diff
@@ -0,0 +1,30 @@
+Function Invoke-Rebase() {
+    <#
+    .SYNOPSIS
+    Convert a number, represented as a sequence of digits in one base, to any other base.
+
+    .DESCRIPTION
+    Implement general base conversion of a number.
+    Given an array of digits represent a number in base "a", convert it and return an array of digits represent the same number in base "b".
+
+    .PARAMETER Digits
+    Array of digits represent the number to be converted.
+
+    .PARAMETER InputBase
+    The original base of the number.
+
+    .PARAMETER OutputBase
+    The base to be converted to.
+
+    .EXAMPLE
+    Invoke-Rebase -Digits @(1, 0, 1 , 0 ,1 ) -InputBase 2 -OutputBase 10
+    return : @(2, 1)
```

Comment:
Yeah I understand what you mean, the reason I did this is because array comparison is not as straight forward in powershell as some other languages, with a simple -eq. So I gotta use compare-object for it. Join is a bit "hacky" way to do, and i suppose might confuse learners who look at the test.
If you don't like it being done this way, I can edit the test cases. I think i might have done this a couple of time in some other exercise as well, because i thought it look a bit cleaner than using compare-object.

Reply:
Cant we just do this:

```pwsh
        $got = Get-PascalsTriangleRows -Count 2
        $want = @(
            @(1),
            @(1, 1)
        )

        $got | Should -Be $want
```

Also I made a request to add you to the exercism organization, which should mean that your prs don't automatically close, if you accepted the invite (and if it was sent).

Reply:
To be honest I didn't like that join method at all, but I had some problem making the test to work properly when testing array so I had to resort to that. I will give it a try and report back soon. 
And I just saw the invitation and accepted it now, thank you. I did leave a msg regarding test cases name, please answer when you have the time, thanks.